ups_modul/inout/inout_gsheet.py

```diff
@@ -3,178 +3,235 @@
-    labels_top_list = ["SUM hours", "wage", "wage - 1%", "tips ratio", "tips", "total"]
-    for x in range(len(labels_top_list)):
-        a1_notion = gspread.utils.rowcol_to_a1(2, days_range+x+2)
-        a2_notion = gspread.utils.rowcol_to_a1(1+anthill.num_of_ants, days_range+x+2 )
-        wks.update_cell(2+anthill.num_of_ants, days_range+2+x, "=SUM({}:{})".format(a1_notion, a2_notion))
-    # the tips cell is calculated manually at the end of the month so it is set to 0
-    wks.update_cell(2+anthill.num_of_ants, days_range+6, "0")
-
-# names for each row
-def names_rows(wks):
-    for keys in anthill_name:
-        wks.update_cell(anthill_name[keys].row, 1, anthill_name[keys].name)
-
-# SUM of hours of the during the month
-def SUM_hours(days_range, wks):
-    for keys in anthill_name:
-        a1_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row,2)
-        a2_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+1)
-        wks.update_cell(anthill_name[keys].row, days_range+2, "=SUM({}:{})".format(a1_notion, a2_notion))
-
-# SUM hours and pay
-def SUM_wage(days_range, wks):
-    for keys in anthill_name:
-        a1_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+2)
-        wks.update_cell(anthill_name[keys].row, days_range+3 ,"=SUM({}*{})".format(a1_notion, anthill_name[keys].pay))
-
-# SUM 99% of the wage
-def SUM_wage99(days_range, wks):
-    for keys in anthill_name:
-        a1_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+3)
-        wks.update_cell(anthill_name[keys].row, days_range+4 ,"=0.99*ROUNDDOWN({};0)".format(a1_notion))
-
-# SUM tips ratio for each row user time to all time
-def SUM_tips_ratio(days_range, wks):
-    a2_notion = gspread.utils.rowcol_to_a1(2+anthill.num_of_ants, days_range+2)
-    for keys in anthill_name:
-        a1_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+2)
-        wks.update_cell(anthill_name[keys].row, days_range+5 ,"={}/{}".format(a1_notion, a2_notion))
-
-# SUM tips based on ratio
-def SUM_tips(days_range, wks):
-    a2_notion = gspread.utils.rowcol_to_a1(2+anthill.num_of_ants, days_range+6)
-    for keys in anthill_name:
-        a1_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+5)
-        wks.update_cell(anthill_name[keys].row, days_range+6 ,"=ROUNDDOWN({}*{};0)".format(a1_notion, a2_notion))
-
-# SUM total of the payment
-def SUM_total(days_range, wks):
-    for keys in anthill_name:
-        a1_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+4)
-        a2_notion = gspread.utils.rowcol_to_a1(anthill_name[keys].row, days_range+6)
-        wks.update_cell(anthill_name[keys].row, days_range+7 ,"=ROUNDDOWN({}+{};0)".format(a1_notion, a2_notion))
-
-# writing the user time in the user cell in the day IN
-def user_time_log(day_in, user_name, hours, wks):
+def names_rows(anthill, wks):
+    '''
+    names from the anthill list in rows
```

Comment:
Function header not clear

Reply:
names in anthill_list is written in first column sorted alphabeticly